gradle.properties

```diff
@@ -1,4 +1,6 @@
 kotlin.code.style=official
 
 GROUP=io.github.detekt.sarif4k
-VERSION=0.3.0-SNAPSHOT
\ No newline at end of file
+VERSION=0.3.0-SNAPSHOT
+
+org.gradle.jvmargs=-XX:MaxMetaspaceSize=512m
```

Comment:
Oh wow, even for this tiny project it needs this much?

Reply:
I got OOM on this project during Dokka processing and applied this workaround: https://github.com/Kotlin/dokka/issues/1405

Decreased to `256m`